### ISSUE TITLE
Send button does not appear or link is not automatically sent when sharing links with talk users from other apps on Android

### ISSUE THREAD

**Issue by maksimaee** [nextcloud/talk-android#4488]
### Steps to reproduce

1. Open an app you want to share a link from (e.g., Instagram, TikTok, etc.).
2. Click the "Share" option for a link and select the Nextcloud Talk app.
3. In the Nextcloud Talk app, choose a user or conversation to share the link with.
4. Notice that:
- The link is pasted into the input field but does not get sent automatically.
- The "Send" button does not appear immediately.

### Expected behaviour

The link should either:
1. Be sent automatically to the selected user/conversation after being shared. (preferred)
2. If manual confirmation is required, the "Send" button should appear immediately.
(This setting can be a configurable option in settings, name: Autosend shared links)

### Actual behaviour

- The link is pasted into the input field but is not sent automatically.
- The "Send" button does not appear until you perform an additional typing action (e.g., press any key, type a letter, or press Enter in the input field).

Action_1
![Image](https://github.com/user-attachments/assets/cecdc808-b33f-4e55-872d-7e5ca37e72b6)
Action_2
![Image](https://github.com/user-attachments/assets/68ab5fe3-4828-456b-8e64-a28a2163386b)
Action_3
![Image](https://github.com/user-attachments/assets/5b7470d1-79f5-4a9c-ae97-3d6babaaa486)
Action_4
![Image](https://github.com/user-attachments/assets/08e69ad1-e5fb-4044-b39c-e69607978787)

Last two steps could have been avoided with automatic link sending. At least the send button should show up right away so that typing action would not be necessary in order to hit the send button.

### Device brand and model

Samsung Galaxy S24 Ultra

### Android version

14

### Nextcloud Talk app version

20.0.5

### Nextcloud server version

30.0.2

### Talk version

20.1.0-rc.1

### Custom Signaling server configured

None

### Custom TURN server configured

None

### Custom STUN server configured

None

### Server log

```bash

```

**Comment by PH03N1X999** [comment]
Yes I was going to post this issue now that someone has done it already, I'm glad to know... Hoping a fix will come soon

**Comment by mahibi** [comment]
can reproduce.

maybe it's related to changes in https://github.com/nextcloud/talk-android/pull/4525 or something (but the PR did not seem to fix it...)
